server/aap/import_text_archive/commands.py

```diff
@@ -63,16 +69,24 @@ class AppImportTextArchiveCommand(superdesk.Command):
         superdesk.Option('--password', '-pwd', dest='password', required=True),
         superdesk.Option('--url_root', '-url', dest='url', required=True),
         superdesk.Option('--query', '-qry', dest='query', required=True),
-        superdesk.Option('--count', '-c', dest='limit', required=False)
+        superdesk.Option('--count', '-c', dest='limit', required=False),
+        superdesk.Option('--direction', '-d', dest='direction', required=False)
     )
 
-    def run(self, start_id, user, password, url, query, limit):
+    BATCH_SIZE = 500
+
+    def run(self, start_id, user, password, url, query, limit, direction):
         print('Starting text archive import at {}'.format(start_id))
         self._user = user
         self._password = password
         self._id = int(start_id)
         self._url_root = url
         self._query = urllib.parse.quote(query)
+        # direction True is forwards
+        self._direction = True
```

Comment:
I would name this _direction_forward, as it is used in other places too. _direction suggest it's values can be 'forward', 'backward'